posydon/popsyn/synthetic_population.py

```diff
@@ -1577,6 +1577,12 @@ def create_transient_population(
                 )
 
             previous = end
+            
+        # it can happen that no systems are selected, in which case nothing has been appended to the file in the loop
+        with pd.HDFStore(self.filename, mode="r") as store:
+            if '/transients/'+transient_name not in store.keys():
+                Pwarn("No systems selected for the transient population!", "NoSystemsSelectedWarning")
```

Comment:
I think you have to add a "NoSystemsSelectedWarning" to the posydonwarning.py file because we don't currently have one.

Reply:
FYI: any not defined warning specified by a string, will default to the base POSYDONwarning. Thus, the warning works, but will be issued as a general POSYDONwarning.

Reply:
Renamed to a general warning for clarity.